app/views/support/Pricing.scala

```diff
@@ -57,7 +57,7 @@ object Pricing {
               s"${discountAmount.pretty} for 1 year, then ${originalAmount.pretty} every year thereafter"
             }
           case OneYear => s"${discountAmount.pretty} for 1 year"
-
+          case _ => ""
```

Comment:
should it be "no discount"? this is if someone applies a discount to a 6Weeks plan?